### DISCUSSION TITLE
add a filtrer about list

### DISCUSSION THREAD

**Discussion by wekerSnail** [community/community#8648] [Other Feature Feedback, Questions, & Ideas]
wo can filter by created list

**Comment by brianlovin** [comment]
Hey @wekerSnail I'm not quite sure I understand what your feedback is about — could you say a bit more about what kind of change or improvement you're hoping for?

**Comment by slmkhanahmed** [comment]
https://addons.mozilla.org/en-US/firefox/addon/git-star/
